DotNetStandardApi/Helpers/StateExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using KoenZomers.Tado.Api.Entities;
```

Comment:
Personally don't see the added value of adding this extension. Why not just use the bool from the State object directly?

Reply:
I prefer the extension method as it lets me ignore the fact that it's a `Nullable<bool>`.

But I can just declare it in my own project(s) 😊.

Reply:
I'd say just use GetValueOrDefault if you don't want it to be a nullable bool :) If you don't mind, I will implement it without the extension to keep it in line with how the rest of the code has been done already. Thanks for this nice contribution nevertheless!